docs/management/admin/response-actions-config.asciidoc

```diff
@@ -25,66 +24,107 @@ preview::[]
++
+- Give the API client the least privilege required to read CrowdStrike data and perform actions on enrolled hosts.
+- Take note of the client ID, client secret, and base URL: you'll need them in later steps when you configure {elastic-sec} components to access CrowdStrike.
+
+. **Install the CrowdStrike integration and {agent}.** Elastic's {integrations-docs}/crowdstrike[CrowdStrike integration]
+ collects and ingests logs into {elastic-sec}.
++
+.. Go to **Integrations**, search for and select **CrowdStrike**, then select **Add CrowdStrike**.
+.. Configure the integration with an **Integration name** and optional **Description**.
+.. Select **Collect CrowdStrike logs via API**, and enter the required **Settings**:
+   - **Client ID**: Client ID for your CrowdStrike API client.
+   - **Client Secret**: Client secret allowing you access to CrowdStrike.
+   - **URL**: The base URL of the CrowdStrike API.
+.. Select the **Falcon Alerts** and **Hosts** sub-options under **Collect CrowdStrike logs via API**.
+.. Scroll down and enter a name for the agent policy in **New agent policy name**. If other agent policies already exist, you can click the **Existing hosts** tab and select an existing policy instead. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+.. Click **Save and continue**.
+.. Select **Add {agent} to your hosts** and continue with the <<enroll-agent,{agent} installation steps>> to install {agent} on a resource in your network (such as a server or VM). {agent} will act as a bridge collecting data from CrowdStrike and sending it back to {elastic-sec}.
+
+. **Create a CrowdStrike connector.** Elastic's {kibana-ref}/action-types.html[CrowdStrike connector] enables {elastic-sec} to perform actions on CrowdStrike-enrolled hosts.
 +
-.Expand for details
+IMPORTANT: Do not create more than one CrowdStrike connector.
++
+.. Go to **Stack Management** → **Connectors**, then select **Create connector**.
+.. Select the CrowdStrike connector.
+.. Enter the configuration information:
+   - **Connector name**: A name to identify the connector.
+   - **CrowdStrike API URL**: The base URL of the CrowdStrike API.
+   - **CrowdStrike Client ID**: Client ID for your CrowdStrike API client.
+   - **Client Secret**: Client secret allowing you access to CrowdStrike.
+.. Click **Save**.
+
+. **Create and enable a rule to generate {elastic-sec} alerts.** (Optional) Create a <<create-custom-rule,custom query detection rule>> to generate {elastic-sec} alerts based on CrowdStrike events and data. Use the index pattern `logs-crowdstrike*`.
++
+NOTE: Do not include any other index patterns.
++
+This gives you visibility into CrowdStrike without needing to leave {elastic-sec}. You can perform supported endpoint response actions directly from alerts that the rule creates, by using the **Take action** menu on the alert details flyout.
```

Comment:
I think _in_ is more appropriate here since the menu is within the scope of the flyout.

```suggestion
This gives you visibility into CrowdStrike without needing to leave {elastic-sec}. You can perform supported endpoint response actions directly from alerts that the rule creates, by using the **Take action** menu in the alert details flyout.
```